tests/Commands/ResultsUpdateCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace RecipeScraperTests\Commands;
+
+use InvalidArgumentException;
+use RecipeScraper\Factory;
+use RecipeScraperTests\UsesTestData;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\VarExporter\VarExporter;
+
+class ResultsUpdateCommand extends Command
+{
+    use UsesTestData;
+
+    protected function configure()
+    {
+        $this->setName('results:update')
+            ->setDescription('Update result files with content from its corresponding HTML file')
+            ->addArgument(
+                'fields',
+                InputArgument::REQUIRED,
+                'A comma separated list of result fields to update'
```

Comment:
Same as in ResultsUpdateAllCommand - I would prefer this as an option with a default value set to all available fields.

Reply:
I see `$properties` in `SchemaOrgMarkup` and`SchemaOrgJsonLd`.
Putting the same list also in this file would be a bit redundant,
How do you find the idea of moving this list to `ScraperInterface` as a `public static $PROPERTIES` and use it in the other places?
```php
$properties = [
        'author',
        'categories',
        'cookingMethod',
        'cookTime',
        'cuisines',
        'description',
        'image',
        'ingredients',
        'instructions',
        'name',
        'notes',
        'prepTime',
        'publisher',
        'totalTime',
        'url',
        'yield',
        'nutrition'
    ];
```